Sources/TuistDependencies/SwiftPackageManager/Utils/PackageInfoMapper.swift

```diff
@@ -429,13 +429,20 @@ extension SourceFilesList {
 }
 
 extension ResourceFileElements {
-    fileprivate static func from(resources: [PackageInfo.Target.Resource], path: AbsolutePath) -> Self? {
+    fileprivate static func from(resources: [PackageInfo.Target.Resource], path: AbsolutePath, excluding: [String]) throws -> Self? {
         let resourcesPaths = resources.map { path.appending(RelativePath($0.path)) }
         guard !resourcesPaths.isEmpty else { return nil }
-        return .init(resources: resourcesPaths.map { absolutePath in
-            let absolutePathGlob = absolutePath.extension != nil ? absolutePath : absolutePath.appending(component: "**")
-            return .glob(pattern: Path(absolutePathGlob.pathString))
-        })
+        return try .init(
+            resources: resourcesPaths.map { absolutePath in
+                let absolutePathGlob = absolutePath.extension != nil ? absolutePath : absolutePath.appending(component: "**")
+                return .glob(pattern: Path(absolutePathGlob.pathString))
+            },
+            excluding: excluding.map {
+                let excludePath = path.appending(RelativePath($0))
+                let excludeGlob = excludePath.extension != nil ? excludePath : excludePath.appending(component: "**")
```

Comment:
I'd add a test to ensure we mapping logic is right and to prevent this from breaking in the future.